### DISCUSSION TITLE
AArch64 binaries in downloads page

### DISCUSSION THREAD

**Discussion by malwareinfosec** [mitmproxy/mitmproxy#7247] [Q&A] [answered]
I checked the [downloads page](https://mitmproxy.org/downloads/#11.0.0/) and was looking for AArch64 binaries. Is there any plan to provide them for future releases?

In the meantime, what is the recommended process to make an AArch64 build myself?

Thanks

**Comment by emanuele-em** [comment]
take a look [here](https://archlinuxarm.org/packages/aarch64/mitmproxy) 🙂

**Comment by mhils** [comment, marked as answer]
https://github.com/mitmproxy/mitmproxy/issues/6756

We already do have arm64 Docker images.

**Reply by malwareinfosec** [reply]
Hey Max, thanks. I did already go down this path with Docker but ran into an issue that I did not encounter with standalone binaries on non ARM devices to run mitmproxy in transparent mode:

``docker run --rm -it -v ~/.mitmproxy:/home/mitmproxy/.mitmproxy -p 8080:8080 mitmproxy/mitmproxy mitmproxy --mode transparent``

![image](https://github.com/user-attachments/assets/38a455b9-a96d-4d9a-95e8-9dd80e9ffd32)

FileNotFoundError(2, 'No such file or directory')

Since I'm unfamiliar with Docker, I found the standalone binaries more straightforward, which is why I wanted to know if they could be available as download without Docker. Or maybe I just have to spend some time and learn about Docker :-)

**Reply by mhils** [reply]
Without having tried it, invoking Docker with `--network=host` may make things work here.

**Reply by malwareinfosec** [reply]
It sure does, thank you!
